### ISSUE TITLE
[Task] Files we couldn't receive - Disable scroll when user prefers-reduced-motion

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#121816]
### Feature

document-status

### Team

bmt-team-2 (Bee’s Knees)

### Issue Type

Task

### Summary

Disable the scroll on the page when the veteran has prefers-reduced-motion settings turned on

#### Issue
Unexpected motion, including scrolling, can be physically harmful for veterans with vestibular diseases, even causing seizures. The scrolling that happens when clicking "Learn about other ways to send your documents" should have a version for users who have the `prefers-reduced-motion` setting on that doesn't scroll, and just "jumps." This should also be true for the pagination scrolling.

Do a search of `vets-website` for `prefers-reduced-motion` and you'll see some examples. It might look something like this in the css:

`
@media (prefers-reduced-motion) {
  .travel-pay-download-loading-icon {
    animation: none;
    transition: none;
  }
}
`

#### Resources
https://developer.chrome.com/docs/devtools/rendering/emulate-css
https://css-tricks.com/introduction-reduced-motion-media-query/
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

### Acceptance Criteria / Definition of Done

- [ ] Any of the scrolling on the page (pagination, the learn about other ways link) should have a `prefers-reduced-motion` version that does not scroll 

### Estimate

None

### Points

3

### Priority

Highest
